lib/licenses.nix

```diff
@@ -347,6 +347,11 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     fullName = "OpenSSL License";
   };
 
+  oraclejavabcl = {
+    fullName = "Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX";
+    url = "http://www.oracle.com/technetwork/java/javase/terms/license/index.html";
```

Comment:
Unfree licenses must have `free = false` to prevent them from being accepted by default.